README.md

```diff
@@ -117,6 +117,20 @@ This setting accepts an array of regular expressions. To match on all warnings,
 DeprecationToolkit::Configuration.warnings_treated_as_deprecation = [//]
 ```
 
+### 🔨 `#DeprecationToolkit::Configuration#allowed_warnings`
+
+It's useful some time to throw the warning from your application code, and treat all warnings as deprecations at the same time
+
+Deprecation Toolkit allows you to configure which warnings should be treated as deprecations and which warnings is allowed to skip at the same time.
+
+This setting accepts an array of regular expressions.
+
+```ruby
+DeprecationToolkit::Configuration.warnings_treated_as_deprecation = [//]
```

Comment:
Why not use a negated regex here instead? Do we need seperate configuration for this?

```
DeprecationToolkit::Configuration.warnings_treated_as_deprecation = [
  /(?!Some useful warning from your code)/
]
```

Reply:
Will it work if I have few different warnings? I wasn't able easily configure this using negated regexp.